### ISSUE TITLE
Accessibility - Keyboard navigation

### ISSUE THREAD

**Issue by Michael-Hanley** [domoritz/leaflet-locatecontrol#268]
Hello, 

Thanks for creating this awesome project! it's been incredibly useful. 

The issue I have been facing is related to the accessibility of the locate control. It is not included in the tab order so it cannot be focused on or acted on with a keyboard.

**Comment by domoritz** [comment]
Thanks for noting the issue. Can you send a pull request to fix it?

**Comment by Michael-Hanley** [comment]
Hello, 

I've opened a PR with a fix. 

Thanks

**Comment by domoritz** [comment]
Will be fixed in https://github.com/domoritz/leaflet-locatecontrol/pull/269
